Src/CSharpier/SyntaxPrinter/CSharpierIgnore.cs

```diff
@@ -1,23 +1,17 @@
 namespace CSharpier.SyntaxPrinter;
 
+using System.Text;
+
 internal static class CSharpierIgnore
 {
     public static bool IsNodeIgnored(SyntaxNode syntaxNode)
     {
-        if (
-            syntaxNode.Parent
-            is not (
-                BaseTypeDeclarationSyntax
-                or BlockSyntax
-                or CompilationUnitSyntax
-                or NamespaceDeclarationSyntax
-            )
-        )
-        {
-            return false;
-        }
-
-        return (Token.HasLeadingComment(syntaxNode, "// csharpier-ignore"));
+        return syntaxNode.Parent
+                is BaseTypeDeclarationSyntax
+                    or BlockSyntax
+                    or CompilationUnitSyntax
+                    or NamespaceDeclarationSyntax
+            && Token.HasLeadingComment(syntaxNode, "// csharpier-ignore");
```

Comment:
Do we want this to also work for 
```
/* csharpier-ignore */
```

Reply:
Maybe only if someone complains? I feel like `//` is pretty standard for c# single line comments